accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/rest/controllers/BasicAccessionHistoryController.java

```diff
@@ -44,4 +46,10 @@ public BasicAccessionHistoryController(HistoryService<MODEL, ACCESSION> service,
         return converter.convert(service.getHistory(accession));
     }
 
+    @RequestMapping(value = "/{accession}", method = RequestMethod.GET, produces = "application/json")
```

Comment:
This method has the same @RequestMapping as above method.

Reply:
sorry forgot to add the verb let me add it , thanks